app/scripts/modules/kubernetes/loadBalancer/details/details.html

```diff
@@ -97,6 +97,17 @@ <h3 select-on-dbl-click>
           </copy-to-clipboard>
         </dd>
       </dl>
+      <dl ng-if="loadBalancer.service.status.loadBalancer.ingress.length">
+        <dt>External IPs</dt>
```

Comment:
You know what, let's call it Ingress to avoid confusion for people familiar with k8s.